Cargo.toml

```diff
@@ -33,4 +33,10 @@ members = [
 opt-level = 3
 
 [patch.crates-io]
+# Some niceties for parsing geojson feature properties.
+# Upstreaming at https://github.com/georust/geojson/pull/155
 geojson = { git = "https://github.com/georust/geojson", branch = "mkirk/try_from" }
+
+# Minimize bandwidth and requests
+# Waiting for release of https://github.com/bjornharrtell/flatgeobuf/pull/93)
+flatgeobuf = { git = "https://github.com/bjornharrtell/flatgeobuf" }
```

Comment:
With these changes it's fairly snappy. It's already faster than downloading and parsing a prepared geojson.

However if we get a much larger data structure, I have some wild ideas for how to make things even faster.

Reply:
I was half-following the discussion in the PRs -- what a detailed, cool contribution to a common lib! I'm sure many other use cases will benefit from this

Reply:
Also, if you're interested, this would be pretty cool work to demo to [cugos](https://cugos.org/meetings/) -- the next meeting should be Jan 20